kfp_notebook/pipeline/_notebook_op.py

```diff
@@ -71,7 +71,7 @@ def __init__(self,
         """
 
         self.notebook = notebook
-        self.notebook_name = self._get_file_name_with_extension(notebook, 'ipynb')
+        self.notebook_name = self._get_file_name_with_extension(notebook)
```

Comment:
If all the method is doing is call basename, we should just cut to the chase.
```suggestion
        self.notebook_name = os.path.basename(notebook)
```

Reply:
However, what if there is no extension here?  Should this be the place where we enforce an extension of either `.ipynb` or `.py`?  In that case, the method would still be useful to house that raising of exception.

Reply:
the notebook in this case is the path of the file and must be valid. In this case, we might tight the validation but I don't believe we should care about the missing extension.